src/commands/config/welcome_slash/info.ts

```diff
@@ -29,10 +29,6 @@ export async function welcomeInfo(interaction: CommandInteraction) {
 
   const configData = configs.data
   if (!configData) {
-    throw new Error(`Failed to get welcome channel response`)
-  }
-
-  if (!configData.welcome_message) {
```

Comment:
Sao chỗ này remove v em

Reply:
em thấy chỉ có case `data=null` chứ ko có `data.welcome = ""` tại bên BE em có default cho nó rồi
với em check case trên trước nó throw luôn ko xuống case dưới response user dc